dev/docs/README.md

```diff
@@ -33,8 +25,7 @@ In addition to the online sites above, Flutter's documentation can be downloaded
 as an HTML documentation ZIP file for use when offline or when you have a poor
 internet connection.
 
-**Warning: the offline documentation files are quite large, approximately 700 MB
-to 900 MB.**
+**Warning: the offline documentation files are quite large, approximately 300MB zipped, 1GB unzipped.**
```

Comment:
Any chance we could use the note block format? I'm not sure it should be a "warning" either.

```
> [!NOTE]
> The offline documentation files are quite large, approximately 300MB zipped, 1GB unzipped.
```